data/core.yaml

```diff
@@ -724,6 +724,13 @@ en:
     add_fields: "Add field:"
     lock:
       suggestion: 'The "{label}" field is locked because there is a Wikidata tag. You can delete it or edit the tags in the "Tags" section.'
+    maxheight:
+      # symbol for meters
+      meter: m
+      # abbreviation of feet
+      foot: ft
```

Comment:
These abbreviations are already present in the YAML file, but only as part of format strings:

https://github.com/openstreetmap/iD/blob/bb56584ce597028a96e28f87410d5f2741d7b4ad/data/core.yaml#L2278

In theory, we could reuse those strings to decide whether to display the unit dropdown to the left or right of the input box, but I don’t know if any language needs that extra complexity.